codespell_lib/_codespell.py

```diff
@@ -278,15 +320,15 @@ def parse_options(args):
                              'corrections. If this flag is not specified or '
                              'equals "-" then the default dictionary is used. '
                              'This option can be specified multiple times.')
-    builtin_opts = '\n- '.join([''] + [
-        '%r %s' % (d[0], d[1]) for d in _builtin_dictionaries])
+    builtin_opts = {d[0]: d[1] for d in _builtin_dictionaries}
     parser.add_argument('--builtin',
                         dest='builtin', default=_builtin_default,
                         metavar='BUILTIN-LIST',
                         help='comma-separated list of builtin dictionaries '
                         'to include (when "-D -" or no "-D" is passed). '
-                        'Current options are:' + builtin_opts + '\n'
-                        'The default is %(default)r.')
+                        'Current options are:' + dict_to_help(builtin_opts) + '\n'  # noqa: E501
+                        'The default is %(default)r.',
+                        ).complete = dict_to_complete(builtin_opts)
```

Comment:
Does this also offer completion of multiple entries e.g. `rare,code`?

Reply:
No, I think user can change `codespell --builtin rare,code` to `codespell --builtin rare --builtin code`.